.github/workflows/rust.yml

```diff
@@ -7,6 +7,9 @@ on:
     # Check if it works with current dependencies (weekly on Wednesday 2:32 UTC)
     - cron: '32 2 * * 3'
 
+env :
+  DATABASE_URL : postgres://postgres:postgres@localhost/fang
```

Comment:
idk if you should add this as a secret

Reply:
it is not needed to add as a secret because it is for testing purposes, everyone should know what db is used for testing.